docker/scripts/init-letsencrypt.sh

```diff
@@ -27,9 +27,9 @@ if [ ! -e "$data_path/conf/options-ssl-nginx.conf" ] || [ ! -e "$data_path/conf/
   echo
 fi
 
-echo "### Creating dummy certificate for $domains ..."
-path="/etc/letsencrypt/live/$domains"
-mkdir -p "$data_path/conf/live/$domains"
+echo "### Creating dummy certificate for ${domains[0]} ..."
```

Comment:
@marccarre 
In the original script, the "domains" variable may contain multiple hostnames (e.g., domains=(example.org www.example.org)).

The linting changes the code so that only the first item is used (I agree that will be the usual case) but then will fail for this special case. 

Is there a way to parametrize/disable this change, or should we change the initial script (in particular the initialisation of the variable ?

Reply:
Otherwise, I'm fine with this PR. Maybe we can put a note/caveat in the doc and open an issue for later ?

Reply:
I may be missing something 🤔, but it looks like the linter correctly raised [`SC2128`](https://github.com/koalaman/shellcheck/wiki/SC2128) here? Indeed, using `$domains` only prints the first value of the array, so the script may have had bugs from the very beginning?

```console
$ cat test_array.sh
#!/bin/bash
set -x
domains=(example.org www.example.org)
echo $domains
echo "$domains"
echo "${domains[0]}"
echo "${domains[@]}"

$ ./test_array.sh
+ domains=(example.org www.example.org)
+ echo example.org
example.org
+ echo example.org
example.org
+ echo example.org
example.org
+ echo example.org www.example.org
example.org www.example.org
```

So, I guess it is a matter of where we want to use the full array, and where we may have bugs and want to iterate over the array?
For example:

1. We may want:
    ```console
    echo "### Creating dummy certificate for ${domains[@]} ..."
    ```
    instead of:
    ```console
    echo "### Creating dummy certificate for $domains ..."
    ```
    in order to print all values?

2. But we may want a `for` loop in places like:
    ```console
    docker-compose -f docker/docker-compose.yml --project-directory . \
      run --rm --entrypoint "\
      rm -Rf /etc/letsencrypt/live/$domains && \
      rm -Rf /etc/letsencrypt/archive/$domains && \
      rm -Rf /etc/letsencrypt/renewal/$domains.conf" certbot
    ```
    (Or maybe we'd want to revisit the script more broadly?)

Am I missing something?
WDYT @pgraverdy?
(If my analysis is correct, happy to revisit this PR to have `for` loops where relevant.)